build.gradle.kts

```diff
@@ -91,7 +91,7 @@ configurations {
 }
 
 tasks.compileJava {
-  dependsOn("openApiGenerateBFF","openApiGenerateP4PAAUTH")
+  dependsOn("openApiGenerateBFF","openApiGenerateP4PAAUTH","openApiGenerateOrganization")
```

Comment:
should this be configured also in the Dockerfile?

Reply:
added